iFixit/react-commerce
# Add Sentry Logging to Frontend Performance

## Overview
We want to add sentry logging for the frontend performance, so this pr introduces changes to do so at the same rate as `ifixit/fixit`. 

## CR
The `pnpm-lock.yaml` changes seemed necessary for `ts` standards, but I'm not sure how to determine issues of changing tons of packages or if I should do this a better way to reduce the code change.

## QA
You'll want to change the value of `tracesSampleRate` in `frontend/sentry.client.config.ts` to `1` to make sure 100% of logging goes to sentry. Then run `pnpm dev` and check if `react-commerce-dev` on sentry is getting performance logs.

Closes: https://github.com/iFixit/ifixit/issues/44070
